### PR TITLE
- Add secure Route functionality

### PR DESCRIPTION
**Pull request by yasnaderi**

### CODE REVIEW

File: src/Illuminate/Routing/Redirector.php [hunk 1]
```diff
@@ -178,6 +178,20 @@ public function route($route, $parameters = [], $status = 302, $headers = [])
...
+    /**
+     * Create a new secure redirect response to a named route.
+     *
+     * @param  string  $route
+     * @param  mixed  $parameters
+     * @param  int  $status
+     * @param  array  $headers
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function ecureRoute($route, $parameters = [], $status = 302, $headers = [])
```
Comment (chu121su12) [inline src/Illuminate/Routing/Redirector.php:190]: You missed the spelling.

Comment (yasnaderi) [reply src/Illuminate/Routing/Redirector.php]: Thanks

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If possible, please consider releasing your code as a package so that the community can still take advantage of your contributions! 

If you feel absolutely certain that this code corrects a bug in the framework, please "**@**" mention me in a follow-up comment with further explanation so that GitHub will send me a notification of your response.

**Comment by yasnaderi** [comment]
@taylorotwell 
The code should be correct and it's just a small fix
Laravel don't have secured redirect with Named Route.
Ex:
In routes/web.php
```php
Route::get("/login", 'App\Http\Controllers\AppUsers@login')->name('MyLogin');
```

in Controller:
```php
return redirect()->route('MyLogin'); // Redirect to HTTP
return redirect()->secure('MyLogin'); // Don't accept Named Route
return redirect()->secure('/login'); // Is the solution without using Named Route

```

**Pull request closed without merging**